tests/misc/process_args.c3

```diff
@@ -1,5 +1,5 @@
 @require_once "std/arithmetic.c3"
-@require_once "std/syscalls.c3"
+@require_once "syscalls.c3"
```

Comment:
Including it as `"syscalls.c3"` is fine for `std` library code, not so great for user code.